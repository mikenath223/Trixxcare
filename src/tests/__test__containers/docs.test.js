import React from 'react';
import withReduxRouter from '../../utils/testUtils';
import DocPage from '../../containers/doc/docPage';


describe('DocPage', function () {
  beforeEach(() => {
    this.props = {
      match: {
        params: {
          name: 'doctors',
        },
      },
    };
  });

  it('should match snapshot', () => {
    const { container } = withReduxRouter(<DocPage {...this.props} />);

    expect(container).toMatchSnapshot();
  });
});
