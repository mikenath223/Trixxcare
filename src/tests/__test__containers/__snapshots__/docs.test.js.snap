// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocPage should match snapshot 1`] = `
<div>
  <div
    class="containerLoad"
  >
    <img
      alt=""
      src="https://www.ecoloxtech.com/images/giphycolor.gif"
    />
  </div>
</div>
`;
